yaml/shared/src/main/scala/org/virtuslab/yaml/internal/load/reader/token/FinalBreak.scala

```diff
@@ -0,0 +1,36 @@
+package org.virtuslab.yaml.internal.load.reader.token
+
+enum FinalBreak(indicator: Char):
```

Comment:
Maybe
```suggestion
enum BlockChompingIndicator(indicator: Char):
```
would be better? Also, that's how the spec is calling it, so it would be more consistent.